packages/i18n-lint/src/verifiers/htmlVerifier.js

```diff
@@ -19,7 +19,7 @@ const noMissingTag = string => {
   const openTags = [];
   const closedTags = [];
 
-  _.forEach(getMatches(string, /<([a-zA-Z]+)>/g, 1), tag => {
+  _.forEach(getMatches(string, /<(\w+)[^/]*>/g, 1), tag => {
```

Comment:
```suggestion
  _.forEach(getMatches(string, /<(\w+)[^\/]*>/g, 1), tag => {
```

Shouldn't we escape the `/` in thee regex ?

Reply:
/shrug
![image](https://user-images.githubusercontent.com/26571211/119498179-a0ad7180-bd65-11eb-997a-dc40609823f4.png)